solr/core/src/resources/ImplicitPlugins.json

```diff
@@ -66,29 +66,13 @@
       "class": "solr.LukeRequestHandler",
       "useParams":"_ADMIN_LUKE"
     },
-    "/admin/system": {
```

Comment:
I think we can't get away with this; the admin UI uses it and perhaps other parts of Solr?  Did you try bringing up Solr with this change and using the admin UI to click on specific cores?